public/app/features/dashboard-scene/settings/annotations/AnnotationSettingsList.tsx

```diff
@@ -0,0 +1,135 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { AnnotationQuery } from '@grafana/data';
+import { getDataSourceSrv } from '@grafana/runtime';
+import { Button, DeleteButton, IconButton, useStyles2, VerticalGroup } from '@grafana/ui';
+import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
+import { ListNewButton } from 'app/features/dashboard/components/DashboardSettings/ListNewButton';
+
+type Props = {
+  annotations: AnnotationQuery[];
+  onNew: () => void;
+  onEdit: (idx: number) => void;
+};
+
+export const AnnotationSettingsList = ({ annotations, onNew, onEdit }: Props) => {
+  const styles = useStyles2(getStyles);
+
+  const onMove = (idx: number, direction: number) => {
+    console.log('todo: onMove');
+  };
+
+  const onDelete = (idx: number) => {
+    console.log('todo: onDelete');
+  };
+
+  const showEmptyListCTA = annotations.length === 0 || (annotations.length === 1 && annotations[0].builtIn);
+
+  const getAnnotationName = (anno: AnnotationQuery) => {
+    if (anno.enable === false) {
+      return (
+        <>
+          <em className="muted">(Disabled) &nbsp; {anno.name}</em>
+        </>
+      );
+    }
+
+    if (anno.builtIn) {
+      return (
+        <>
```

Comment:
Why do we need these fragments?

Reply:
Don't reckon we do, I have removed them.